src/lib/y2storage/boot_requirements_strategies/bls.rb

```diff
@@ -28,6 +31,22 @@ def initialize(*args)
         super
       end
 
+      def self.bls_bootloader_proposed?
```

Comment:
As commented above, I would move this method to `Analyzer`.

Reply:
Moved...